core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalCheckpointThread.java

```diff
@@ -121,9 +122,9 @@ public long getNextSequenceNumber() {
   public void run() {
     try {
       runInternal();
-    } catch (RuntimeException e) {
-      LOG.error("{}: Failed to run journal checkpoint thread, crashing.", mMaster.getName(), e);
-      throw e;
```

Comment:
FYI, only this part conflicted during auto cherry-pick.
Apperantly exc type was diferent in 1.8.